src/CMakeLists.txt

```diff
@@ -60,12 +60,6 @@ endforeach()
 
 source_group("Header Files\\Public" FILES ${PUBLIC_HEADERS})
 
-include(GNUInstallDirs)
-
-set(BIN_INSTALL_DIR ${CMAKE_INSTALL_BINDIR})
-set(LIB_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR})
-set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR})
```

Comment:
Are these variables not used? Or does rocm-cmake already set them?

Reply:
They are unused in rocALUTION and—as far as I can tell—they don't control anything in cmake or rocm-cmake.

Reply:
As per last discussion/comments on changes for modules, we are using CMAKE flags directly in module and so these flags are not used.
 
If I understand correctly, the  rocm-cmake changes (CMAKE FLAGS based)  updated in https://github.com/RadeonOpenCompute/rocm-cmake/pull/95, https://github.com/RadeonOpenCompute/rocm-cmake/pull/96